parts/res/layout/thermal_list_item.xml

```diff
@@ -48,15 +48,27 @@
 
         <Spinner
             android:id="@+id/app_mode"
-            android:layout_width="wrap_content"
+            android:layout_marginTop="2dp"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content" />
 
     </LinearLayout>
 
+    <ImageView
+        android:id="@+id/touch"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:paddingStart="16dp"
+        android:paddingEnd="16dp"
+        android:scaleType="centerInside"
+        android:src="@drawable/ic_touch"
```

Comment:
You missed this.

Reply:
Looks more of a remnant from the Touch Profiles feature we had, can just safely get rid of this drawable entry

(irrelevant FYI, touch control settings aren't really needed to add as it can automatically go to the max sensitivity without this settings)